packages/devops/scripts/ci/testE2e.sh

```diff
@@ -4,7 +4,7 @@ DIR=$(dirname "$0")
 DEPLOYMENT_URL=$(${DIR}/determineDeploymentUrl.sh)
 if curl --output /dev/null --silent --head --fail $DEPLOYMENT_URL; then
     echo "Deployment for ${CI_BRANCH} exists, running e2e tests"
-    ssh ubuntu@$DEPLOYMENT_URL "cd tupaia && yarn workspace @tupaia/${PACKAGE} test-e2e"
+    ssh ubuntu@$DEPLOYMENT_URL "cd tupaia && yarn workspace @tupaia/${PACKAGE} test-e2e --ciBuildId ${CI_BUILD_ID}"
```

Comment:
Is it cypress or codeship that provides `CI_BUILD_ID` as an env var? If it's codeship, convenient that it matches cypress terminology!

Reply:
It's Codeship! Cypress devs must had Codeship Pro in mind - it's the only [CI/CD platform](https://docs.cypress.io/guides/guides/parallelization#CI-Build-ID-environment-variables-by-provider) whose name matches the cypress option! 😆

Reply:
Ha, nice!